fairscale/optim/oss.py

```diff
@@ -105,9 +105,9 @@ def __init__(
         self._device = list(self.per_device_params.keys())[0]
         self.buckets: Dict[torch.device, List[Bucket]] = {}
 
-        # if torch.cuda.is_available() and self.world_size <= torch.cuda.device_count():
-        #     broadcast_buffer_size = 0
-        #     logging.warning("Assuming single node job, bucketing is disabled")
+        if torch.cuda.is_available() and self.world_size <= torch.cuda.device_count():
```

Comment:
this was a mistake, commented out to force a test of the bucketing and it slipped in master